features/eyedropper.yml

```diff
@@ -0,0 +1,5 @@
+name: EyeDropper
+description: The EyeDropper API creates an eyedropper tool that allows users to select a color from their screen.
+spec: https://wicg.github.io/eyedropper-api/#eyedropper-interface
+compat_features:
+  - api.EyeDropper
```

Comment:
Can you list all 4 BCD keys that are listed by `npm run traverse | grep EyeDropper` in BCD? (Something like this is usually the starting point for me.)

Reply:
Is `api.EyeDropper.secure_context_required` really needed? I thought this kind of usage requirement was handled at a different level in BCD, not as a compat key.

Reply:
Oh, that changed the versions from 95 to 96. That doesn't seem right to me, the feature not working over HTTP isn't essential in any way. So let's exclude the key, after all.